### DISCUSSION TITLE
16 APR Dev Session Notes

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1208] [General]
Notes from 16 April Session let me know if I missed or misrepresented anything: 

- Around the room to discuss what folks are working/ thinking about.
     -  **Mesa Visualization challenges, install and way forward** :  In order to update the javascript front end, the choice was made to download directly instead via npm. This make it easier for the user (who doesn't need to install npm) but places burden on the developers (who must update in multiple spaces) and make reproducibility harder.  This is seen as a temporary fix with a long term fix developed in the context of the Mesa development strategy #1288. One recommended way forward is to create a separate repo `Mesa-Visualization 0.1.0`.
     - **EOL 3.6**: We need to signal Python 3.6 is no longer supported @jackiekazil plans to place in the release notes 

- Review [Quartzsite release](https://github.com/projectmesa/mesa/milestone/25) Looking at the milestones my gut says many of these are addressed or partly addressed, but could be wrong.
     - #1142 @EwoutH ? or transfer to @rht 
     - #1165 @jackiekazil 
     - #1282 @tpike3 owes @rht a review
     - #1210 @tpike3 (if no new contributor claims) 

- Mesa-Geo update @wang-boyu 
    - Continues to do great work and gave a shout out to @rht for his awesome support
    - @jackiekazil coordinate with @Corvince  to release Mesa-Geo PyPI authority

- Establish May milestones 
    -  #1270 @tpike3 needs to review (will also close #1088)
    -  #1076 @jackiekazil     

- Discuss improvements to reach Mesa 1.0 
    - #808

**Comment by GaelLucero** [comment]
added pull request #1278 for discussion. Work previously presented in collaboration with @aiwhoo

**Reply by GaelLucero** [reply]
Made an additional commit (8a2b3feba690b8f6568ab0592cd57c6073b09aaf) to pull request #1278 ready for further review and discussion

**Comment by aiwhoo** [comment]
I'm sorry to ask, but is the meeting 800est or 700est given the time zone differences and day light savings time?

**Reply by rht** [reply]
By comparing it with the other timezones listed, I think it is 8:00 EDT. We should just call it 8:00 ET.

**Reply by tpike3** [reply]
Oh dang I though it was the reverse (stupid daylight savings), however @rht's solution is much more elegant so updated

**Comment by wang-boyu** [comment]
@tpike3 you probably @ the wrong person with the same name as mine who also happens to be a PhD student lol

And I was wondering why there was no notification of being mentioned here : )

**Reply by tpike3** [reply]
Well crap...I have fixed... sorry about that

**Reply by wang-boyu** [reply]
No worries!

**Comment by Corvince** [comment]
@tpike3 Is it correct that you edited this thread to change it from the dev meeting announcement to the dev session notes? I didn't receive a notification for your edit and stumbled upon the notes rather accidently, because I was really looking forward to the notes. 

I think in the future it would be better to instead comment the notes, because edits don't seem to cause notifications.

**Reply by tpike3** [reply]
@Corvince Good to know, thanks! In the future I will make the notes separate so there is a notification.
